### PR TITLE
wr-321: Adjust layout to wide screens

### PR DESCRIPTION
**Pull request by drMangust**
Task - [Adjust layout to wide screens#321](https://github.com/BinaryStudioAcademy/bsa-2023-writorium/issues/321)

image
image

### CODE REVIEW

**Review by andanf-e — DISMISSED** [review]
LGTM, but checked avatar placeholder initials on profile page, and letters looks pretty small, can we increase the font-size for it on profile?

**Comment by drMangust** [comment]
> LGTM, but checked avatar placeholder initials on profile page, and letters looks pretty small, can we increase the font-size for it on profile?

image

**Comment by drMangust** [comment]
image

File: frontend/src/pages/auth/components/auth-layout/styles.module.scss [hunk 1]
```diff
@@ -46,7 +45,8 @@
 }
 
 .scrollableFormWrapper {
-  flex-basis: 553px;
+  flex-basis: 33%;
+  min-width: 500px;
```
Comment (andanf-e) [inline frontend/src/pages/auth/components/auth-layout/styles.module.scss]: Not sure if min-width: 500 would be suitable for narrow screens

Comment (drMangust) [reply frontend/src/pages/auth/components/auth-layout/styles.module.scss]: [85b212f](https://github.com/BinaryStudioAcademy/bsa-2023-writorium/pull/372/commits/85b212f5e0c376445d779dfbaf157519586135e3)

**Review by andanf-e — APPROVED** [review]

**Pull request merged**